app/browser/tabs.js

```diff
@@ -188,6 +188,10 @@ const sendAboutDetails = (tabId, type, value, shared = false) => {
   if (aboutTabs[tabId] && !aboutTabs[tabId][type].get(value)) {
     const tab = webContentsCache.getWebContents(tabId)
     if (tab && !tab.isDestroyed()) {
```

Comment:
Should we also consider logging these lines / conditions?:
- `tab` is null or destroyed
- `aboutTabs[tabId]` is null?
- aboutTabs[tabId] is initialized at `ipcMain.on(messages.ABOUT_COMPONENT_INITIALIZED`